x-pack/plugin/sql/jdbc/src/main/java/org/elasticsearch/xpack/sql/jdbc/JdbcConfiguration.java

```diff
@@ -208,7 +215,8 @@ public boolean indexIncludeFrozen() {
     }
 
     public static boolean canAccept(String url) {
-        return (StringUtils.hasText(url) && url.trim().startsWith(JdbcConfiguration.URL_PREFIX));
+        return (StringUtils.hasText(url) &&
```

Comment:
minor: I would trim the url and use a local var to avoid trimming it potentially twice.